MODULE.bazel

```diff
@@ -4,6 +4,7 @@ module(
     compatibility_level = 1,
 )
 
+bazel_dep(name = "rules_android", version = "0.5.0")
```

Comment:
There's a newer version out already, might be worth while just using that directly. 6.0.0 should also decouple us from needing the .bazelrc flags for enabling access to the native providers.

Reply:
Sounds good, I upgraded here: d23ab768513eda7da0d6908d137807bf4892b024